templates/kubernetes/4/rancher-compose.yml

```diff
@@ -22,6 +22,9 @@ etcd:
         increment: 1
         max: 3
         min: 1
+    metadata:
```

Comment:
Entrypoint script needs this value. If not present, it just happens to have the same effect as if it were 1 (but any time min_scale > 1, we absolutely need it)